app/elements/single-record/single-record.html

```diff
@@ -194,7 +212,20 @@
       },
       getAllItemKeys: function(item) {
         if(item.details) {
-          return _.keys(item.details);
+          var sortedKeys = [];
+          var returnKeys = [];
+          if(item.details._sortedKeys) {
+            sortedKeys = item.details._sortedKeys
```

Comment:
We're not currently enforcing any code style rules so this is not a strict requirement, but it would be nice if we could follow some general style guidelines like ending lines with semicolons (lines 218 and 228) and using consistent spacing (like the space between the if and the open parentheses on line 224).  Thanks!